metaflow/plugins/kfp/kfp_constants.py

```diff
@@ -3,7 +3,9 @@
 # Defaults for running MF on KFP
 import os
 
-BASE_IMAGE = "hsezhiyan/metaflow-zillow:2.1"
+# Link to package:
+# https://github.com/zillow/metaflow/pkgs/container/metaflow%2Fmetaflow-zillow
+BASE_IMAGE = "ghcr.io/zillow/metaflow/metaflow-zillow:2.1"
```

Comment:
How are we building this image?

Reply:
It's probably built manually for now.

Reply:
Hmm how is it pushed to that Docker repo? Is that manually from @hsezhiyan's machine then too? Might be worthwhile trying to document that process quickly too if possible.

Reply:
Hey @alexlatchford , I've documented that [here](https://github.com/zillow/metaflow/blob/feature/kfp/metaflow/plugins/kfp/Dockerfile#L7), let me know if you want anything else.

Reply:
🤦  makes sense, thanks!